---
template: post.html
title: "Comparison of Single-cell Long-read and Short-read Transcriptome Sequencing of Patient-derived Organoid Cells of ccRCC: Quality Evaluation of the MAS-ISO-seq Approach<br/>Natalia Zajac, Functional Genomics Center Zurich"
description: 'Zurich Seminars in Bioinformatics'
date: 2024-04-04
authors:
  - "@mbaudis"
links:
  - '[Functional Genomics Center Zurich](https://fgcz.ch)'
---

* 12:15 UZH Irchel Y55-l-06/08 and ZOOM Call

**Abstract** Single-cell RNA sequencing is used in profiling gene expression differences between cells. Short-read sequencing platforms provide high throughput and high-quality information at the gene-level, but the technique is hindered by limited read length, failing in providing an understanding of the cell heterogeneity at the isoform level. This gap has recently been addressed by the long-read sequencing platforms that provide the opportunity to preserve full-length transcript information during sequencing.

To objectively evaluate the information obtained from both methods, we sequenced four samples of patient-derived organoid cells of clear cell renal cell carcinoma and one healthy sample of kidney organoid cells on Illumina Novaseq 6000 and PacBio Sequel IIe. For both methods, for each sample, the cDNA was derived from the same 10x Genomics 3' single-cell gene expression cDNA library. Here we present the technical characteristics of both datasets and compare cell metrics and gene-level information.

We show that the two methods largely overlap in the results but we also identify sources of variability which present a set of advantages and disadvantages to both methods.