---
title: 'Shinichi Sunagawa - Metagenomics'
template: blog_post.html
date: 2020-10-20
authors:
  - "@SuShiAtGit"
pdf_file_name:
links:
  - '<a href="/UZH-BIO390/course-material/2020-10-20___Shinichi-Sunagawa__Metagenomics__UZH-BIO390-HS20-lecture-06.pdf" target="_blank">[2020 lecture slides]</a>'
  - '<a href="http://baudisgroup.org/storage-ext/2020-10-20___Shinichi-Sunagawa__Metagenomics__UZH-BIO390-HS20-lecture-06-recording.mov" target="_blank">[lecture video recording] (ca. 150MB .mov)</a>'
---

## Metagenomics
#### Shinichi Sunagawa (ETHZ)

**Abstract:**

Microorganisms are numerically dominant on Earth and drive the cycling of energy, elements and matter. Thanks to advances in high-throughput DNA sequencing technologies and computational power, microbial communities can now be studied without the need to cultivate them in a laboratory setting. Essential tasks in studying microbial communities include the identification and quantification of their member taxa and the pair-wise compositional comparison of different microbial communities.
<!--more-->
To this end, typically PCR-amplified fragments of universal taxonomic marker genes are sequenced and quantified to generate taxon abundance tables. These tables can then be analyzed to determine the composition and diversity of, and the differences between microbial communities. By random (shotgun) sequencing of total community DNA samples, it is, although still challenging, possible to reconstruct microbial draft genomes by a series of steps including metagenome assembly and binning. This way, it is possible to access genomic information of organisms that cannot be readily studied under laboratory conditions. Such approaches have revealed an immense, underexplored taxonomic and gene functional diversity of our microbial world.

**Lerning objectives:**
- You can formally describe the composition and diversity of, and the differences between microbial communities
- You understand the process, value and challenges of reconstructing microbial genomes from metagenomes

<!--more-->
