---
title: 'Proteomics'
template: post.html
authors:
  - "@mbaudis"
date: 2019-10-29
pdf_file_name: 2019-10-29___Katja-Baerenfaller__Proteomics__UZH-BIO390-HS19-lecture-06.pdf    # name of PDF (no path) somewhere in "assets"; auto-linked
links:  # one or more formatted html links
  - '<a href="/UZH-BIO390/">[UZH-BIO390]</a>'
  - '<a href="/UZH-BIO390/categories/people.html">[Course Lecturers]</a>'
---

#### Katja Baerenfaller

In proteomics one of the important bioinformatics tasks is to generate lists of reliably identified peptides and proteins in mass spectrometry-based experiments. For this, amino acid sequences are assigned to measured tandem mass spectra. The quality of the peptide spectrum assignments are scored and criteria are applied that allow to distinguish the good from the bad hits and to estimate the quality of the dataset.

<!--more-->

In the context of this lecture, you will need to learn and understand:  

* what information can be gained in a proteomics experiment
* whatâ€™s the principle of assigning an amino acid sequence to a tandem mass spectrum
* how de novo and database-dependent peptide identifications work
* one way of how the accuracy of peptide identifications can be estimated
* how the number of wrong hits in a dataset can be estimated
* current proteomic approaches


